sct.py

```diff
@@ -472,7 +472,7 @@ def send_email(test_id=None, email_recipients=None, logdir=None):
     if not reporter:
         LOGGER.warning("No reporter found")
     else:
-        test_results['nodes'] = get_running_instances_for_email_report(test_id)
+        test_results['nodes'] = get_running_instances_for_email_report(test_results['test_id'])
```

Comment:
how can `test_results['test_id']` be different from `test_id`?

Reply:
by Alex
> when you provide test-id it works ok, thats why i missed this.
but if you set only logdir, this means that sct.py have to find out latest test run
and get the testrun logdir
in this case the test_id stay none, and by this reason, there is no any instances running

Reply:
it command runs with logdir parameter only. this means we search latest run. and test-id stays None.